### PR TITLE
Detect new brands: Ziox, iDroid, Aoson, Cloudfone, Oyyu, Primux, iMars, Logic, Vsmart and Detect new apps: Viber, Siri, LinkedIn, Twitter, and Detect devices for existing brands: Vertu, Amigoo, Aligator, Google, Phicomm, Alcatel, Realme, TLS, Xiaomi, Umax, Sony, Lava, OnePlus, Yu, Amazon, Cherry Mobile, ZTE, Mobiistar, MyPhone, Medion, Lenovo, Qilive, Ulefone, Highscreen

### LINKED ISSUE #6366: Brand Model Detection Suggestions 2020-09-10
brand | model | deviceType | sample | done
-- | -- | -- | -- | --
realme | C15 | smartphone | Mozilla/5.0 (Linux; Android 10; RMX2180 Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 (Mobile; afma-sdk-a-v203019038.203019038.0) |  :heavy_check_mark:
realme | C12 | smartphone | Mozilla/5.0 (Linux; Android 10; RMX2189 Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 |  :heavy_check_mark:
ZTE | Zmax 3 | smartphone | Mozilla/5.0 (Linux; Android 10; dandelion Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/81.0.4044.138 Mobile Safari/537.36 (Mobile; afma-sdk-a-v203019035.203019035.0) |  :heavy_check_mark:
Xiaomi | Redmi 9A | smartphone | Mozilla/5.0 (Linux; Android 10; M2006C3LG Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 (Mobile; afma-sdk-a-v203019035.203019035.0) |  :heavy_check_mark:
LOGIC | X5 Lite | smartphone | Dalvik/2.1.0 (Linux; U; Android 5.1; LOGIC_X5_LITE Build/LMY47D) |  :heavy_check_mark:
Cherry Mobile | Flare S8 | smartphone | Mozilla/5.0 (Linux; Android 9; Flare S8 Build/PPR1.180610.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/84.0.4147.111 Mobile Safari/537.36 |  :heavy_check_mark:
Mobiistar | C1 Lite | smartphone | Mozilla/5.0 (Linux; Android 8.1.0; C1 Lite Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/68.0.3440.91 Mobile Safari/537.36 |  
Cloudfone | Thrill Boost 3 | smartphone | Mozilla/5.0 (Linux; Android 7.0; Thrill Boost 3 Build/NRD90M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/61.0.3163.98 Mobile Safari/537.36 |  :heavy_check_mark:
Meitu | M6 | smartphone | Dalvik/2.1.0 (Linux; U; Android 9; M6 Build/P00610) |  
OnePlus | Nord (5G) | smartphone | Dalvik/2.1.0 (Linux; U; Android 10; AC2003 Build/QKQ1.200412.002) | :heavy_check_mark:
Vsmart | Bee 3 | smartphone | Dalvik/2.1.0 (Linux; U; Android 9; Bee 3 Build/PPR1.180610.011) |  
Lava | A97 IPS | smartphone | Mozilla/5.0 (Linux; Android 6.0; A97 IPS Build/MRA58K; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/55.0.2883.91 Mobile Safari/537.36 |  :heavy_check_mark:
Alcatel | 1V (2020) | smartphone | Mozilla/5.0 (Linux; Android 10; 5007A Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 (Mobile; afma-sdk-a-v203019035.203019035.0) | :heavy_check_mark: 
Vsmart | Joy 1 | smartphone | Dalvik/2.1.0 (Linux; U; Android 9; Joy 1+ Build/PKQ1.190414.001) |  :heavy_check_mark: 
Lava | Z60 | smartphone | Mozilla/5.0 (Linux; Android 7.0; Z60 Build/NRD90M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/64.0.3282.137 Mobile Safari/537.36 |  
Amazon | Fire HD 8 (2020) | smartphone | Mozilla/5.0 (Linux; Android 9; KFONWI Build/PS7315; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/75.0.3770.143 Safari/537.36 | :heavy_check_mark: 
Alcatel | 3 (2020) | smartphone | Mozilla/5.0 (Linux; Android 10; 5029E Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/84.0.4147.125 Mobile Safari/537.36 | :heavy_check_mark:   
Vsmart | Active 1 | smartphone | Dalvik/2.1.0 (Linux; U; Android 9; Active 1+ Build/PKQ1.190118.001) |:heavy_check_mark:   
Oppo | F1w | smartphone | Dalvik/2.1.0 (Linux; U; Android 5.1.1; F1w Build/LMY47V) |  
Kudae | K101 | smartphone | Mozilla/5.0 (Linux; Android 9; K101 Build/PPR1.180610.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 |  
Micromax | YU ACE | smartphone | Mozilla/5.0 (Linux; Android 8.1.0; YU5014 Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/69.0.3497.100 Mobile Safari/537.36 |:heavy_check_mark:   
HTC | Wildfire CDMA | smartphone | Dalvik/2.1.0 (Linux; U; Android 8.1.0; Bee Build/O11019) |  
Mobiistar | X1 Notch | smartphone | Mozilla/5.0 (Linux; Android 8.1.0; X1 Notch Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/69.0.3497.100 Mobile Safari/537.36 | :heavy_check_mark:   
Lava | Z61 | smartphone | Mozilla/5.0 (Linux; Android 8.1.0; Z61 Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/70.0.3538.110 Mobile Safari/537.36 |  
MyPhone | myA17 | smartphone | Mozilla/5.0 (Linux; Android 9; myA17 Build/PPR1.180610.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/74.0.3729.136 Mobile Safari/537.36 |:heavy_check_mark: 
Mobiistar | E1 Selfie | smartphone | Mozilla/5.0 (Linux; Android 8.1.0; E1 Selfie Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/66.0.3359.126 Mobile Safari/537.36 |:heavy_check_mark:   
OnePlus | Nord (5G) | smartphone | Mozilla/5.0 (Linux; Android 10; AC2001 Build/QKQ1.200412.002; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/81.0.4044.138 Mobile Safari/537.36 |:heavy_check_mark: 
UMiDIGI | A7 Pro | smartphone | Mozilla/5.0 (Linux; Android 10; A7 Pro Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 |  
Alcatel | 3X (2019) | smartphone | Mozilla/5.0 (Linux; Android 9; 5048U Build/PPR1.180610.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 |:heavy_check_mark:   
Doogee | X5 | smartphone | Dalvik/2.1.0 (Linux; U; Android 6.0; X5 Build/MRA58K) |  
Lava | Z50 | smartphone | Mozilla/5.0 (Linux; Android 8.1.0; Z50 Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/70.0.3538.110 Mobile Safari/537.36 |  
Lava | Z60s | smartphone | Mozilla/5.0 (Linux; Android 8.1.0; Z60s Build/O11019; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/70.0.3538.110 Mobile Safari/537.36 |  
Alcatel | 1C | smartphone | Mozilla/5.0 (Linux; Android 8.1.0; 5003A Build/OPM2.171019.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 |:heavy_check_mark:   
Sony | Xperia 5 | smartphone | Mozilla/5.0 (Linux; Android 10; J8210 Build/55.1.A.9.75; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/85.0.4183.81 Mobile Safari/537.36 578b7a00-b3e0-4988-8e61-5feec3840c7c | :heavy_check_mark:

### PR DESCRIPTION
**Pull request by sanchezzzhak**
this PR should be merged after https://github.com/matomo-org/device-detector/pull/6365

### CODE REVIEW

**Comment by sanchezzzhak** [comment]
I stop adding new detections to this PR and start a new branch.
You can check.

**Comment by sgiehl** [comment]
@sanchezzzhak  Thanks again for creating all the PRs. I'll send you an invite for write access. That way you should be able to review and merge pull request from others.

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -13050,9 +13112,14 @@ Vernee:
...
 Vertu:
-  regex: 'Vertu[ ]?([a-z0-9]+)'
+  regex: 'Vertu[ ]?([a-z0-9]+)|ASTER P[/;) ]'
   device: 'feature phone'
-  model: '$1'
+  models:
+    - regex: 'ASTER P[/;) ]'
+      device: 'smartphone'
+      model: 'Aster P'
+    - regex: 'Vernee[ _-]([a-z0-9_-]+)'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml:13121]: Shouldn't that be `Vertu`? Seems there actually was no test for a Vertu device. Maybe we could add one with e.g. this user agent: `Vertu Signature/2.0 (08w31) IMEI/XXXXXXXXXXXXXXX Profile/MIDP-2.0 Configuration/CLDC-1.1`

Comment (sanchezzzhak) [reply regexes/device/mobiles.yml:13121]: You can't find out the model without the first IMEI characters

Comment (sgiehl) [reply regexes/device/mobiles.yml:13121]: Then we can use `Vertu Signature/2.0 (502.06) IMEI/355711020080019 Profile/MIDP-2.1 Configuration/CLDC-1.1`
Nevertheless I think `Vernee` is a typo in the regex, isn't it?

Comment (sanchezzzhak) [reply regexes/device/mobiles.yml]: Yes that's right `Vernee` is a typo

**Review by sgiehl — APPROVED** [review]

**Pull request merged**